client/src/panels/panel_declarations/results/drr_summary.js

```diff
@@ -180,6 +181,95 @@ const StatusGrid = props => {
 };
 
 
+
+class PercentageViz extends React.Component {
+  constructor(props){
+    super(props);
+    const { counts } = this.props;
+
+    const all_ids = _.keys(counts);
+
+    this.state = {
+      selected: _.filter(all_ids, id => id !== "future"), // default to no future
+    };
+  }
+
+  render(){
+    const { counts } = this.props;
+    const { selected } = this.state;
+
+    const data = _.chain(counts)
+      .toPairs()
+      .map(pair => ({label: result_statuses[pair[0]].text, value: pair[1], id: pair[0]}))
+      .value();
```

Comment:
Patterns like this are a useful case for array destructuring, to avoid having to access by index/to add clarity
i.e. `.map(pair =>` to `.map( ([key, value]) =>`

Alternatively, when mapping an object, the second argument of the iterate func becomes is the key. So, this could be written as
```
const data = _.map(
  counts,
  (value, key) => ({
    label: result_statuses[key].text,
    id: key,
    value
  })
);
```

Reply:
:+1: